@here/harp-datasource-protocol/test/ExprTest.ts

```diff
@@ -56,12 +56,14 @@ describe("Expr", function() {
     describe("#fromJson", function() {
         describe("ref operator support", function() {
             const baseDefinitions: Definitions = {
-                color: { type: "color", value: "#ff0" },
-                number: { type: "number", value: 123 }
+                color: "#ff0",
+                number: { type: "number", value: 123 },
+                boolean: true
```

Comment:
we need to support the case where type is optional, e.g.
                `number: { value: 123 },`            
                `width: { value: ["number", 123] },`
should also work.

Reply:
Ups. Fixed. Last patch contains support for `{value: 123}`